src/utils/regionService.tsx

```diff
@@ -39,16 +43,26 @@ const regionListAPI = async (projectId: string) => {
       Authorization: API_HEADER_BEARER + credentials.access_token
     }
   });
+  if (resp.status === 403) {
+    toast.error(`403 Permission denied`, toastifyCustomStyle);
```

Comment:
service calls shouldn't be firing off toasts and setting react state since it should be ui agnostic, could you move the toast and state setting to the dropdown?  You can update useRegion to return an error state if that's what you are looking for.

Reply:
reverted config changes.